src/main/java/org/arkecosystem/client/api/Api.java

```diff
@@ -1,9 +1,8 @@
-package org.arkecosystem.client.api.two;
+package org.arkecosystem.client.api;
 
-import org.arkecosystem.client.api.AbstractAPI;
 import org.arkecosystem.client.http.Client;
 
-public class Two extends AbstractAPI {
+public class Api extends AbstractAPI {
```

Comment:
`AbstractAPI` is superfluous now. Would merge it with `Api`. And `Connection<Api>` becomes then `Connection`, related to other comment.